### ISSUE TITLE
NVDA doesn't announce button names

### ISSUE THREAD

**Issue by jswiderski** [ckeditor/ckeditor5#10247]
## 📝 Provide a description of the improvement

_How the feature works now and what you'd like to change_?  
  
**This problem is related to Chrome based browsers only**

NVDA screen reader gives different results in different browsers for toggle buttons:

*   In Firefox it has been properly announcing button names for all toggle buttons e.g. "Bold toggle button not pressed".
*   In Chrome based browsers is has just been announcing "toggle button not pressed" without specifying button names.

We should check whether there is any aria attribute we can add so that names are properly announced for these types of buttons.  
  
\----  
**This problem is related to all browsers**

When dealing with split buttons e.g. bulleted list, highlight or numbered list, the menu part is just announced as "Menu button collapsed sub-menu" and "Expanded" one it is opened. There is no information to which button this submenu belongs to.

We should check whether it is possible to add any aria attribute to improve that.  
The same happens when using JAWS so either this is how it has to be or this is editor a11y problem in general.

\----

**This problem is related to all browsers**

For menu buttons like font family it is not announced when menu is opened. The button is read as "Menu Font-family button sub-menu" but if you press down arrow nothing is announced. 

We should check whether it is possible to add any aria attribute to improve that.  
The same happens when using JAWS so either this is how it has to be or this is editor a11y problem in general.

## 📃 Other details

*   Browser: Chrome, New Edge, Opera
*   OS: Windows
*   CKEditor version: 29.0.0
*   Installed CKEditor plugins: N/A

---

If you'd like to see this improvement implemented, add a 👍 reaction to this post.

**Comment by bbenjamin** [comment]
This may be the same as what was reported here Splitbutton toggle no tooltip / no AT label. If this is the issue that remains open, the issue name should probably be updated as it specifies NVDA + "button names" but it's more accurate to say all screenreaders with Splitbuttons - something that is made clear in the issue summary.

**Comment by mabryl** [comment]
A connected issue is that NVDA (and other screen readers, e.g. Windows Narrator) don't announce the names of the button when you hover the mouse over them.
